pom.xml

```diff
@@ -605,6 +606,7 @@
         <configuration>
           <showDeprecation>true</showDeprecation>
           <contextPath>/jenkins</contextPath>
+          <compatibleSinceVersion>${hpi-plugin.compatibleSinceVersion}</compatibleSinceVersion>
```

Comment:
Fortunately, maven considers blank values as `null` (that is, just as not specified).
MEXEC-104: http://mojo.10943.n7.nabble.com/jira-Created-MEXEC-104-Cannot-pass-empty-argument-to-exec-goal-td17960.html